index.html

```diff
@@ -4,10 +4,112 @@
+        <li class="desktop-menu"><a href="#">WIA Campain</a></li>
+      </ul>
+    </nav>   
+  </header>
+  
+  <section class="home" id="home">
+    <h1 class="page-title">WOMEN IN ANALYTICS <span></span>CONFERENCE <span></span>2021</h1>
+    <span class="moto">"An Analytics Conference. Featuring Women. For Everyone."</span>
+    <p class="intro">
+      The WIA conference brings together a global community of professionals, and academia while providing a platform for women to lead the conversations around the advancements of analytical research, development, and applications.
+    </p>
+    <span class="date">2021.07.21(WED) ~ 23(FRI)</span>
+    <span class="place">@ Conference Hall, Columbus, Ohio</span>
+  </section>
+
+  <section class="main">
+    <h2>Main Program</h2>
+    <div class="red-underline centered"></div>
+    <ul class="programs">
+      <li>
+        <span class="program-icon"> <i class="far fa-handshake"></i></span>
+        <h3>Career Fair</h3>
+        <p>Network with companies looking for talent in analytics and data science.</p>
+      </li>
+      <li>
+        <span class="program-icon"> <i class="fas fa-archway"></i></span>
+        <h3>Expo Hall</h3>
+        <p>Visit sponsoring companies at the expo.  Stay tuned for more details!</p>
+      </li>
+      <li>
+        <span class="program-icon">  <i class="fas fa-network-wired"></i></span>
+        <h3>Networking Breaks</h3>
+        <p>These breaks are specifically tailored to for meeting new contacts.</p>
+      </li>
+     
+      <li>
+        <span class="program-icon"> <i class="fas fa-couch"></i></span>
+        <h3>Networking Lounge</h3>
+        <p>Spacious lounge is a quiet and comfortable spot to grab a coffee, host a quick meeting.</p>
+      </li>
+      <li>
+        <span class="program-icon"> <i class="far fa-comments"></i></span>
+        <h3>Meet the Speakers</h3>
+        <p>Meet with the experts to make connections, discuss talks or get career advice. </p>
+      </li>
+    </ul>
+
+    <a href="#" class="see-btn">SEE THE WHOLE PROGRAM</a>
+    <a href="#" class="btn">Join WIA Conference 2021</a>
+  </section>
+
+  <section class="speakers">
+    <h2 class="centered">Featured Speakers</h2>
+    <div class="red-underline"></div>
+    <ul class="speaker-list" id="speaker-list">
+      
+    </ul>
+    <button type="button" class="more-btn" id="more-btn">MORE <i class="fas fa-chevron-down"></i></button>
+  </section>
+
+  <section class="partners centered" id="partner">
+    <h2>Partner</h2>
+    <div class="red-underline"></div>
+    <ul class="partner-list">
+      <li><img src="./images/partners/skype.png" alt="Skype"></li>
+      <li><img src="./images/partners/ebay.png" alt="Ebay"></li>
+      <li><img src="./images/partners/paypal.png" alt="PayPal"></li>
+      <li><img src="./images/partners/linkedin.png" alt="Linkedin"></li>
+      <li><img src="./images/partners/amazon.png" alt="Amazon"></li>
+    </ul>
+  </section>
```

Comment:
- [ ] You have a horizontal overflow in both pages on mobile screens. Please fix this.
- [ ] In this section, on Desktop screens, the `show more` button shouldn't be displayed. This should only happen on mobile screens. Kindly fix it. You can do this in the media queries by using a `display: none;`.
- [ ] Also, you are missing the image shadow to the top left corner of the images in this section. Kindly add it.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124140845-b33e5900-da3d-11eb-8dac-78fd819901c9.png) | ![image](https://user-images.githubusercontent.com/49621279/124140954-cf41fa80-da3d-11eb-99bd-4682eecf1920.png)